src/styles/normalize.scss

```diff
@@ -0,0 +1,24 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+
+  --background-properties: no-repeat center;
+  --main-color: #060b35;
+  --accent-color: #00acdc;
+  --secondary-color: #616070;
```

Comment:
Let's use `scss` variables and move them into a separate file. And don't forget to use them

Reply:
> Let's use `scss` variables and move them into a separate file. And don't forget to use them

Not fixed